style.css

```diff
@@ -409,4 +424,103 @@ ul {
 #aboutme-list {
   margin: 0;
   padding: 0;
-}
\ No newline at end of file
+}
+
+#contact {
+  background-color: var(--white);
+}
+
+.contact-bg-form {
+  background-color: var(--primary-1);
+  padding-top: 10%;
+  padding-bottom: 10%;
+  border-radius: 100px 0 0 0 / 100px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./Icons/contact\ form\ shapes-mobile.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 100%;
+  padding-top: 19%;
+  padding-right: 8%;
+  padding-left: 8%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+}
+
+.contact-form .inputs,
+.inputs-msg {
+  width: 100%;
+  height: 48px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background-color: #fff;
+  margin: 20px 0;
+}
+
+.contact-form .inputs-msg {
+  height: 150px;
+}
+
+.contact-form input,
+textarea {
+  display: flex;
+  width: 80%;
+  height: 18px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: var(--color3);
+  margin: 16px 15px 36px 15px;
+  border: none;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] For your form to look much better and adhere to that of the `Figma` design kindly consider the below suggestions as it going to help you do just that. Currently, your form looks okay but you included some of the styles in the wrong place and due to that you need to make the suggested changes to enhance its look to help create a good user experience.
  -[ ] Consider removing the `height` and `margin` you have set on your input as it's not necessary and instead add the `styles` you have on `.contact-form .inputs, .inputs-msg` except the `margin: 20px 0` as those styles should rather be included here. Please make sure that your style matches that of the `Figma` design. See below for more details:
  
  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="373" alt="Screenshot 2023-07-20 at 18 42 56" src="https://github.com/abdullah97-pro/Portfolio-Mobile-Project/assets/107192076/9d4c2f65-cae4-43a5-8f3e-4b2d2d598747">
</td>
<td>
<img width="373" alt="Screenshot 2023-07-20 at 18 06 57" src="https://github.com/abdullah97-pro/Portfolio-Mobile-Project/assets/107192076/49c1c92b-e3d9-4251-b5e7-b28955e53017">
</td>
</tr>
</table>